sdks/python/apache_beam/runners/direct/transform_evaluator.py

```diff
@@ -529,16 +530,19 @@ def start_bundle(self):
     self._counter_factory = counters.CounterFactory()
 
     # TODO(aaltay): Consider storing the serialized form as an optimization.
-    dofn = pickler.loads(pickler.dumps(transform.dofn))
+    dofn = transform.dofn
```

Comment:
<!--new_thread; commit:7549b5c2ebe2ae47af9066eaf97364a27e828ab5; resolved:0-->
We had this line here to ensure that the DoFn was serializable, so that a user would not hit any issues when running later on remote runners.  Can you describe the reason for this change?